_pages/release-strategies.md

```diff
@@ -0,0 +1,115 @@
+
+It's impossible to completely de-risk a release. Developing a plan ahead of time addressing how you will approach fan incident will
+enable your team to focus on fixing the issue as soon as possible. Once you have a response plan, you should conduct 
+[incident type drills](/security/incident-response-drills/#example-incident-response-drills) so that your team is well-practiced in
+what do if something goes wrong.
+
+#### Develop Training Strategies in Advance
+
+Recommendation: Do user research and develop training strategies ahead of time to help users ease into the new system.  
+
+It can be jarring as a user to one day have a completely new system. Spending some time before release working with critical users of the
+system to understand the common pitfalls a user may experience or uncover will help you develop training materials to address those issues 
+(or make the system more intuitive.) Releasing training materials for users to look at before the release will create better familiarity and
+make users less wary or uncomfortable with the big change.
+
+#### Prepare Customer Support Response Templates
+
+Recommendation: Develop templates for expected support requests to have consistent messaging and advice to users.
+
+Launches often have expected support requests, such as: 
+* "Why doesn't the system do X?"
+* "This is broken."
+* "I can't find Z."
+
+Preparing responses ahead of time allows the customer support team to give the same suppport to anyone who a predictable issue, allowing them
+to focus their energies on unexpected support needs. It also helps unsure a better customer experience for users.
+
+### Big Bang Release Factors
+
+There are some factors that make partners more attached to big bang releases. There are some strategies you can deploy to try to 
+convince partners to pivot to a smaller, more iterative release strategy.
+
+#### Partner Doesn't Want to Pick a Cohort of Controlled Users
+
+Understanding why a partner doesn't want to single out some controlled users is important. They may be nervous about creating inequity (or
+a perception of inequity) that might have some polical blowback. Or perhaps they feel overwhelmed about the effort required to narrow down
+to a subset of users, and so feel it's simpler to release to all users at once.
+
+*Mitigation strategies:*
+* Work with your partner to fully understand their fears. 
+* Frame this step as the first of many interactions with their different user groups, and use that framing to help them better engage and 
+understand their users. Having a more mature idea of their user groups will help them have confidence to stand behind their decisions better.
+
+#### A Long, Arduous ATO Process
+
+Partners have a valid fear of the ATO process, and may be nervous that they must repeat the ATO process every time a change is made to the
+system. That perspective easily lends itself to fear of iterative releases. 
```

Comment:
I can't recall where, but we've written a lot before about modular ATOs or iterative ATOs and such. It might be worth digging those up.